packages/cli/src/lib/quire/index.js

```diff
@@ -122,9 +123,16 @@ async function initStarter (starter, projectPath, options) {
   setVersion(projectPath, quireVersion)
 
   /**
-   * Write starter name and version to VERSION_FILE
+   * Write quire-11ty, cli, starter name and version to VERSION_FILE
    */
-  const versionInfo = { 'quire-11ty': quireVersion, starter: { path: starter, version: starterVersion } }
+  const versionInfo = { 
+    cli: packageConfig.version,
+    'quire-11ty': quireVersion, 
+    starter: { 
+      path: starter,
+      version: starterVersion
+    }
```

Comment:
this looks reasonable, would we ever want to write this as follows?
```suggestion
    starter: '<path>@<starterVersion>'
```